src/App.tsx

```diff
@@ -1,10 +1,56 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import { Todo } from './types/Todo';
+import { Error } from './types/Error';
+import { FilterTodos } from './types/FilterTodos';
+import { getTodos } from './api/todos';
 import { UserWarning } from './UserWarning';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer';
+import { ErrorNotification } from './components/ErrorNotification';
 
-const USER_ID = 0;
+const USER_ID = 6550;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState(Error.NONE);
+  const [filterBy, setFilterBy] = useState(FilterTodos.ALL);
+
+  useEffect(() => {
+    const getTodosFromServer = async () => {
+      try {
+        setErrorMessage(Error.NONE);
+        const todosFromServer = await getTodos(USER_ID);
+
+        setTodos(todosFromServer);
+      } catch (error) {
+        setErrorMessage(Error.LOAD);
+      }
+    };
+
+    getTodosFromServer();
+  }, []);
+
+  const countTodos = todos.length;
```

Comment:
you can pass juts `todos.length` to next component

Reply:
![image](https://user-images.githubusercontent.com/114435233/224838136-a53387f0-a5c3-47ef-9141-c93dec17ec15.png)

Reply:
I use it in two places so I will leave it as is